src/components/molecules/UserStatusManager/UserStatusManager.tsx

```diff
@@ -68,7 +70,7 @@ export const UserStatusManager: React.FC<UserStatusManagerProps> = ({
   );
 
   return (
-    <div className="UserStatusManager">
+    <div className={`UserStatusManager ${className}`}>
```

Comment:
Yes, but it will be too much work. I need to create styles for this component, QuestionInput, EntranceInput and I don't really think it's worth it. But we should definitely do this for our staging branch.

If you have a strong opinion about this, I will change it.